content/id/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Disrupsi
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+Petunjuk ini diperuntukkan bagi pemilik aplikasi yang ingin membuat aplikasi yang memiliki ketersediaan yang tinggi, sehingga dibutuhkan untuk mengerti jenis-jenis Gangguan yang dapat terjadi pada Pods.
+
+Petunjuk ini juga diperuntukkan bagi administrator kluster yang ingin melakukan berbagai tindakan otomasi pada kluster, seperti pembaruan dan *autoscaling* kluster.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Disrupsi yang Disengaja dan Tidak Disengaja
+
+Pods tidak akan terhapus hingga sesuatu (orang maupun *controller*) menghancurkan mereka atau ada kesalahan perangkat keras maupun perangkat lunak yang tidak dapat dihindari.
+
+Kita menyebut kasus-kasus yang tidak dapat dihindari sebagai *disrupsi yang tidak disengaja* terhadap aplikasi. Beberapa contohnya adalah sebagai berikut:
```

Comment:
Nit: no need for italic in disrupsi yang disengaja

Reply:
The original does italicize them to emphasize them the first time they are mentioned in this guide. I think we better make it italic to put some emphasis, or we can also change it to bold.

Reply:
let's make it bold for emphasize, since we usually do italic for imported english words